src/core/Akka.Cluster.Tests.MultiNode/SurviveNetworkInstabilitySpec.cs

```diff
@@ -426,11 +425,36 @@ public void A_Network_partition_tolerant_cluster_must_down_and_remove_quarantine
                 }, _config.Third);
                 EnterBarrier("quarantined");
 
+                RunOn(() =>
+                {
+                    Thread.Sleep(2000.Milliseconds());
+
+                    // not be down
+                    var secondUnique = Cluster.State.Members.SingleOrDefault(m => m.Address == GetAddress(_config.Second) && m.Status == MemberStatus.Up);
+                    if (secondUnique == null)
+                    {
+                        // TODO: fail("Unexpected removal of quarantined node")
```

Comment:
This should be some sort of assertion fail. Couldn't figure out the api for that in Akka.NET, yet.

Reply:
You can just use Fluent Assertions. Lookin at the code from original test:

```csharp
var secondUniqueAddress = Cluster.State.Members.SingleOrDefault(m => m.Address == GetAddress(_config.Second));
secondUniqueAddress.Should().Not.BeNull(because: "2nd node should stay visible");
secondUniqueAddress.Status.Should().Be(MemberStatus.Up, because: "2nd node should be Up");
```